docs/book/src/reference/labels_and_annotations.md

```diff
@@ -0,0 +1,48 @@
+**Supported Labels:**
+
+
+|label     |note     |
+|:--------:|:--------:|
+| cluster.x-k8s.io/cluster-name| this label is set on machines linked to a cluster and external objects(bootstrap and infrastructure providers) |
+| topology.cluster.x-k8s.io/owned| this label is set on all the object which are managed as part of a ClusterTopology. |
+|topology.cluster.x-k8s.io/deployment-name | this label is set on the generated  MachineDeployment objects to track the name of the MachineDeployment topology it represents. |
+| cluster.x-k8s.io/provider| this label is set on components in the provider manifest. this label allows to easily identify all the components belonging to a provider. the clusterctl tool uses this label for implementing provider's lifecycle operations. |
+| cluster.x-k8s.io/watch-filter | this label can be applied to any Cluster API object. Controllers which allow for selective reconciliation may check this label and proceedwith reconciliation of the object only if this label and a configured value is present. |
+| cluster.x-k8s.io/interruptible|this label is used to mark the nodes that run on interruptible instances. |
+|cluster.x-k8s.io/control-plane | this label is set on machines or related objects that are part of a control plane. |
+| cluster.x-k8s.io/set-name| this label is set on machines if they're controlled by MachineSet. |
+| cluster.x-k8s.io/deployment-name| this label is set on machines if they're controlled by MachineDeployment. |
+| machine-template-hash| this label is applied to Machines in a MachineDeployment containing the hash of the template. |
+<br>
+
+
+
+**Supported Annotations:**
+
+|annotation     | note     |
+|:--------:|:--------:|
+| topology.cluster.x-k8s.io/managed-field-paths   |  this annotation is used to store the list of paths managed by the topology controller. changes to those paths will be considered authoritative.  |
```

Comment:
Could you note that this is deprecated? Same for other labels which are listed as deprecated.

Reply:
Did it.